packages/aws-cdk/test/context-providers/test.vpcs.ts

```diff
@@ -44,6 +44,8 @@ export = nodeunit.testCase({
     // THEN
     test.deepEqual(result, {
       vpcId: 'vpc-1234567',
+      vpcCidrBlock: '10.0.0.0/16',
```

Comment:
I'm all in favor of this, but be aware there is a concurrent PR changing the VPC provider, and these changes will have to be integrated across PRs: https://github.com/aws/aws-cdk/pull/4544